modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/routes/ContextRoutes.scala

```diff
@@ -149,27 +159,55 @@ class ContextRoutes(contexts: Contexts[Future], contextQueries: FilterQueries[Fu
           val filter     = filterFrom(deprecatedOpt, None, querySettings.nexusVocBase) and publishedExpr(publishedOpt)
           implicit val _ = (id: ContextId) => contexts.fetch(id)
           (pathEndOrSingleSlash & authenticateCaller) { implicit caller =>
-            contextQueries.list(filter, pagination, None, sort).buildResponse(fields, base, prefixes, pagination)
+            sort match {
+              case SortList.Empty =>
```

Comment:
I would rather have a class which contains both, the `ElasticQueries ` and `SparqlQueries` which depending on the filter, sorting, pagination and so on, decides to choose the elastic indexer or the sparql indexer. The decision to use one or the other can trickier than just if sort query params are defined or not